engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/SelectGameScreen.java

```diff
@@ -241,33 +170,16 @@ private void loadGame(GameInfo item) {
         }
     }
 
-    public boolean isLoadingAsServer() {
+    private boolean isLoadingAsServer() {
         return universeWrapper.getLoadingAsServer();
     }
 
-    public void setUniverseWrapper(UniverseWrapper wrapper) {
+    void setUniverseWrapper(UniverseWrapper wrapper) {
```

Comment:
Probably another access modification we don't need (probably the same for `isLoadingAsServer()` ?)